api/src/main/java/au/org/aodn/nrmn/restapi/validation/validators/global/formatted/Method3QuadratsSum.java

```diff
@@ -0,0 +1,44 @@
+package au.org.aodn.nrmn.restapi.validation.validators.global.formatted;
+
+import au.org.aodn.nrmn.restapi.model.db.StagedRowError;
+import au.org.aodn.nrmn.restapi.validation.StagedRowFormatted;
+import au.org.aodn.nrmn.restapi.validation.validators.base.BaseGlobalFormattedValidator;
+import cyclops.control.Validated;
+import lombok.val;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Method3QuadratsSum extends BaseGlobalFormattedValidator {
```

Comment:
Can't see where this s instantiated - do we need to add it to the global format checks for RLS/ATRC or common